pyaedt/modeler/object3dlayout.py

```diff
@@ -348,6 +348,224 @@ def enabled(self, status=True):
         ]
         return self.change_property(args)
 
+    @property
+    def solderball_enabled(self):
+        """Check if Solderball is enabled.
+
+        Returns
+        -------
+        bool
+        """
+        if self._part_type_id not in [0, 4, 5]:
+            return False
+        component_info = str(list(self.m_Editor.GetComponentInfo(self.name))).replace("'", "").replace('"', "")
+        if "sbsh=Cyl" in component_info or "sbsh=Sph" in component_info:
+            return True
+        return False
+
+    @property
+    def die_enabled(self):
+        """Check if the die is enabled. Valid for IC only.
+
+        Returns
+        -------
+        bool
+        """
+        if self._part_type_id not in [0, 4, 5]:
+            return False
+        component_info = str(list(self.m_Editor.GetComponentInfo(self.name))).replace("'", "").replace('"', "")
+        if "dt=1" in component_info or "dt=2" in component_info:
+            return True
+        return False
+
+    @property
+    def die_type(self):
+        """Get the die type.
```

Comment:
```suggestion
        """Die type.
```

Reply:
I don't think a property needs to have a Returns section. I could be wrong though. According to Alex, it should be a noun description (no "Get" verb).